jerryscript-mbed-ble/BLEJS.h

```diff
@@ -157,10 +209,29 @@ class BLEJS {
         }
     }
 
+    void onDataWrittenCallback(const GattWriteCallbackParams *params) {
+        // see if we know for which char this message is...
+        typedef std::map<GattCharacteristic*, jerry_value_t>::iterator it_type;
+        for(it_type it = write_callbacks.begin(); it != write_callbacks.end(); it++) {
+            if (it->first->getValueHandle() == params->handle) {
+                if (jerry_value_is_function(it->second)) {
+                    const jerry_value_t args[1] = {
+                        getJsValueFromCharacteristic(it->first)
+                    };
+
+                    // @todo, this_obj is wrong
+                    jerry_call_function(it->second, this_obj, args, 1);
```

Comment:
Makes sense - it might be something worth looking into in the future, but for the time being this should be fine.